.gitignore

```diff
@@ -1 +1,12 @@
 .ipy*
+geone/__pycache__/
```

Comment:
These directories should not be here if geone is not installed "locally", but yes, we can add them in .gitignore...